pkg/apis/workspaces/v1alpha1/volumeComponent.go

```diff
@@ -15,4 +15,9 @@ type Volume struct {
 	// +optional
 	// Size of the volume
 	Size string `json:"size,omitempty"`
+
+	// +optional
+	// Ephemeral volumes are not stored persistently across restarts. Defaults
+	// to false
+	Ephemeral bool `json:"ephemeral,omitempty"`
```

Comment:
I just wonder if it makes sense to use `temporary` here instead of avoid confusing between emptyDir and ephemeral volumes (which is in alpha but still) https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#volume-v1-core

Reply:
I originally had `persistent` here -- we switched to ephemeral because that's the k8s verbiage in [docs](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/)

Reply:
I see that `Ephemeral` and `EmptyDir` are types of Ephemeral storage in the K8s.
So, I'm fully OK with using Ephemeral without implementation details, and then if needed - we could use like attributes to specify user desired implementation.